tools/hicup/hicup_truncater.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="hicup_truncater" name="Truncater" version="0.1.0">
+    <macros>
+        <import>hicup_macros.xml</import>
+    </macros>
+    <expand macro="requirements_hicup" />
+    <expand macro="stdio" />
+    <command><![CDATA[
+        export FONTCONFIG_FILE=\$CONDA_PREFIX/etc/fonts/fonts.conf &&
+        export FONTCONFIG_PATH=\$CONDA_PREFIX/etc/fonts/ &&
```

Comment:
I fixed this in the conda package of fontconfigs at conda-forge and made a PR. Let's see if they accept it.